src/goto-programs/add_race_assertions.cpp

```diff
@@ -154,7 +162,8 @@ void add_race_assertions(
       }
 
       // now add assignments for what is written -- reset
-      forall_rw_set_entries(e_it, rw_set) if(e_it->second.w)
+      forall_rw_set_entries(
+        e_it, rw_set) if(e_it->second.w || e_it->second.deref)
```

Comment:
The formatting here is awful to read. We should do something about that with the clang-formatter. This is a general comment, has nothing to do with this PR.